pipelines/shopify/incremental_loading_helpers.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime, timedelta, timezone
+from dateutil import parser
```

Comment:
please import `pendulum` from `from dlt.common import pendulum` and use it
str(pendulum.now()) - to iso format
pendulum.parse(str(pendulum.now())) from iso format. please remove your own parsing

Reply:
Done.
Thanks!